samples/openthread/libraries/README.rst

```diff
@@ -0,0 +1,6 @@
+.. _ot_lib_sample:
```

Comment:
need better doc

Reply:
Sure, I will update it when the technical part of this task is settled.
In case we plan to add some defines or anything like that I will put them here as well.